app/views/admin/facilities/index.html.erb

```diff
@@ -28,12 +28,12 @@
       <div class="card">
         <div class="row">
           <div class="col-8">
-            <h3><span class="heading-label">Facility group</span><%= facility_group.name %></h3>
+            <h3><span class="heading-label"><%= t("facility_group").capitalize %></span><%= facility_group.name %></h3>
           </div>
 
           <% if current_admin.accessible_facility_groups(:manage).find_by_id(facility_group) %>
             <div class="col-4 text-right">
-              <strong><%= link_to 'Edit group', edit_admin_facility_group_path(facility_group), class: "btn btn-sm btn-outline-primary" %></strong>
+              <strong><%= link_to "Edit #{t("facility_group")}", edit_admin_facility_group_path(facility_group), class: "btn btn-sm btn-outline-primary" %></strong>
```

Comment:
Would suggest capitalizing the facility group to keep it consistent with the "+ Add Upazila" button at the top of the page

Reply:
No. We should be using "Sentence case" all over the dashboard @claudiovallejo